ticketutil/servicenow.py

```diff
@@ -0,0 +1,410 @@
+        """
+        try:
+            logging.info('Rewriting CC list')
+            if isinstance(user, str):
+                user = [user]
+
+            fields = {'watch_list' : ', '.join(user)}
+            params = self._create_ticket_parameters(fields)
+            self.s.headers.update(self.headers_post_)
+            r = self.s.put(self.ticket_rest_url, data=params)
+            r.raise_for_status()
+            self.ticket_content = r.json()['result']
+            logging.info('CC list rewritten for {0}'
+                         .format(self.ticket_id))
+        except requests.RequestException as e:
+            logging.error('Failed to rewrite CC list')
+            return False
+
+    def remove_cc(self, user):
+        """
+        Removes user(s) from cc list.
+        :param user: A string representing one user's email address, or a list
+        of strings for multiple users.
+        :return:
+        """
+        try:
+            logging.info('Removing user(s) from CC list')
+            watch_list = self.ticket_content['watch_list'].split(',')
+            watch_list = [item.strip() for item in watch_list]
+            if isinstance(user, str):
+                user = [user]
+            for item in user:
+                if item in watch_list:
+                    watch_list.remove(item)
+
+            fields = {'watch_list' : ', '.join(watch_list)}
+            params = self._create_ticket_parameters(fields)
+            self.s.headers.update(self.headers_post_)
+            r = self.s.put(self.ticket_rest_url, data=params)
+            r.raise_for_status()
+            self.ticket_content = r.json()['result']
+            logging.info('User(s) removed from CC list of {0}'
+                         .format(self.ticket_id))
+        except requests.RequestException as e:
+            logging.error('Failed to remove user(s) from CC list')
+            return False
+
+    def _prepare_ticket_fields(self, fields):
+        """
+        Makes sure each key value pair in the fields dictionary is in
+        the correct form.
+        :param fields: Ticket fields.
+        :return: fields: Ticket fields for the ticketing tool.
+        """
+        for key, value in fields.items():
+            if key in ['opened_for', 'operating_system', 'category', 'item',
+                        'severity', 'hostname_affected', 'opened_by_dept']:
+                fields['u_{}'.format(key)] = value
+                fields.pop(key)
+            if key == 'topic':
+                fields['u_topic_reportable'] = value
+                fields.pop(key)
+            if key == 'email_from':
+                fields['u_email_from_address'] = value
+                fields.pop(key)
+        return fields
+
+
+def devops_one_url(server, table, sys_id):
+    """
+    Creates DevOps One URL of the existing ticket
+    """
+    return '{server}/pnt/?id=ticket&sys_id={sys_id}&table={table}'.format(
+            server=server, sys_id=sys_id, table=table)
```

Comment:
The problem is that ticketutil uses self.project, but for Service Now itself table makes more sense, but I have renamed it to `self.project`.